### ISSUE TITLE
Choose the best logo!

### ISSUE THREAD

**Issue by sarahgaiser** [JPsiInVR/organisation#128]
Please let Dominique know what you thing of his artwork. Comment below which logo you prefer and why 😄 

number | 1 | 2 | 3 | 4 | 5 | 6
--------|---|---|---|---|---|---
logo | <img src="https://user-images.githubusercontent.com/69592687/123299307-ab505780-d519-11eb-8b2f-4e6815f40241.jpg" width="250"> | <img src="https://user-images.githubusercontent.com/69592687/123299325-b30ffc00-d519-11eb-943f-70ba854a7b0e.jpg" width="250">  | <img src="https://user-images.githubusercontent.com/69592687/123299395-c28f4500-d519-11eb-9c4b-784f4e5313ff.jpg" width="250"> | <img src="https://user-images.githubusercontent.com/69592687/123299341-b7d4b000-d519-11eb-9d4c-9e11cadb4b7b.jpg" width="250"> | <img src="https://user-images.githubusercontent.com/69592687/123299372-bdca9100-d519-11eb-9e51-4af556cbea95.jpg" width="250"> | <img src="https://user-images.githubusercontent.com/69592687/123299386-c02ceb00-d519-11eb-82cd-2576e64c46f6.jpg" width="250">

**Comment by Layfully** [comment]
I like both 3 and 4. I think 4th one would be better if it was oriented the same way as 3rd one.

**Comment by MPogwizd** [comment]
I like the second and sixth the most. I feel like, the original concept was great, and those are improved variants. The sixth might look better when finalized.

**Comment by Ursuss21** [comment]
Every logo is absolutely amazing, but for our project's sake I think that the 3rd one works the best.

**Comment by tristen-shields** [comment]
I like 3 the best because it does the best at arranging the letters in a way where you can tell it is 'JUNO'.

**Comment by s6nadavi** [comment]
I agree with Robert in the sense that every logo is pretty amazing! Personally, I prefer the third logo. It's compact and you can instantly recognize the name of our project (as Tristan already mentioned) :)

**Comment by aaceituno01** [comment]
I agree with most of you, personally I prefer the 3rd one.

**Comment by PSSteinmann** [comment]
"I like both 3 and 4. I think 4th one would be better if it was oriented the same way as 3rd one." I agree  on  this one
